Orange/widgets/visualize/owboxplot.py

```diff
@@ -322,6 +291,11 @@ def set_data(self, dataset):
 
         self.dataset = dataset
         if dataset:
+            sample_at = 1000
+            if len(self.dataset) > sample_at:
+                rows = np.random.choice(len(self.dataset), size=sample_at, replace=False)
+                self.sample = Table.from_table_rows(self.dataset, rows)
```

Comment:
from_table_rows morda ni treba tukaj klicati. Zakaj ne `self.dataset[indeksi]`? Ker kar dobis, itak ni navadna tabela (glej vrstico dalje). :)

Reply:
Ja to sem najprej napisal, potem pa sem mislil da je lepse ce to funkcijo poklicem, ce ravno ze obstaja.
Mogoce je res bolje kar z indeksi (__getitem__ tako ali tako klice from_table_rows).
Lahko pa ze ko to popravljam probam kar v Tablu naredit nek sampling, ali je to stvar zasvoj branch?